client/state/ui/post-type-list/test/reducer.js

```diff
@@ -22,8 +20,6 @@ describe( 'reducer', () => {
 		expect( reducer( undefined, {} ) ).to.have.keys( [
```

Comment:
How difficult would it be to rewrite this test to use the Jest `expect` API here and remove the `chai` import?

Reply:
I didn't want to make a gigantic PR out of this. But I'll definitely follow-up in another PR 👍

Reply:
Done as part of #45293.